apps/web/app/helpers/date.ts

```diff
@@ -84,11 +84,27 @@ export const formatDateTimeString = (dateTimeString?: string) => {
 		const formattedDate = `${day} ${monthParsed} ${year}`;
 		const timeString = time.split('.')[0];
 		const timeArray = timeString.split(':');
-		const amPm =  parseInt(timeArray[0]) >= 12 ? 'PM' : 'AM' 
+		const amPm = parseInt(timeArray[0]) >= 12 ? 'PM' : 'AM';
 		const formattedTime = `${timeArray[0]}.${timeArray[1]} ${amPm}`;
 
 		return `${formattedDate}, ${formattedTime}`;
 	}
 
 	return '';
 };
+
+export const calculateRemainingTime = (
```

Comment:
@desperado1802 use `moment` library to do this type of calculation. We are already using this library in our project.